packages/ckeditor5-table/tests/ui/colorinputview.js

```diff
@@ -218,6 +218,26 @@ describe( 'ColorInputView', () => {
 				expect( inputView.value ).to.equal( 'bar' );
 			} );
 
+			it(
+				`when the color input value is set to one of defined colors, but with few additional white spaces,
+				should use its label as the text input value`,
```

Comment:
```suggestion
			it( 'should use its label as the text input value (normalizing color input value to match defined colors)', () => {
```

Reply:
I was thinking about something like that. Shorter description. But if you guys already agreed on that I'm :man\_shrugging: on this :)

Reply:
OK... I saw PK was writing that long descriptions already. For me those are too long but as above :man\_shrugging: . Let's keep that tests consistent.